packages/core/src/lib/test/project-configurator.ts

```diff
@@ -23,15 +23,18 @@ export type FileTreeContentType =
   | string[]
   | SheriffConfigContent;
 
-export type FileTree = { [key: string]: FileTreeContentType };
+// eslint-disable-next-line @typescript-eslint/consistent-indexed-object-style
```

Comment:
Sticking to the type here would cause another linter error?

Reply:
Exactly. It will say it is a circular reference

Reply:
Okay, I'm not really happy with ignoring eslint errors, but maybe this is something we can cover in a follow-up issue